experiments/20160805-mutagenesis/20160907-pb2-pt1-mytaq.md

```diff
@@ -0,0 +1,29 @@
+## Goals
+
+- Produce low-mutation PCR products of PB2 part 1 with MyTaq
+
```

Comment:
For more context, can you add a reference to issue #13 (https://github.com/ericmjl/protein-systematic-characterization/issues/13#issuecomment-245415630)? This will help us understand (later) why we decided to do a low-mutation PCR product.